packages/api/test/stripe.spec.js

```diff
@@ -103,3 +105,171 @@ describe('StripeCustomersService', async function () {
     assert.equal(customer2ForUser2.id, customerForUser2.id, 'should return same customer for same userId2')
   })
 })
+
+describe('StripeSubscriptionsService', async function () {
+  it('can saveSubscription between prices using real stripe.com API', async function () {
+    // ensure stripeSecretKey and use it to construct stripe
+    const stripeSecretKey = process.env.STRIPE_SECRET_KEY
+    if (!stripeSecretKey) {
```

Comment:
that's cool - for some reason it never occurred to me that you could skip a test dynamically like that; I've always just added it to the `it` call if I knew it wasn't going to pass for some reason.

Reply:
(FWIW me too, I think I copypastad the `it.skip()` otherwise I might not have thought about it)